README.md

```diff
@@ -12,6 +12,10 @@ The application suite consists of three core components:
 
 ### Releases
 
+[Release 5.2.5](https://github.com/google/earthenterprise/releases/tag/5.2.5-714.41)
+* [Release Notes](http://www.opengee.org/geedocs/5.2.5/answer/7160006.html)
```

Comment:
we probably should start using https for opengee.org links

Reply:
changed